apps/src/dev/app.html.ejs

```diff
@@ -12,10 +12,6 @@
   <link rel="stylesheet" type="text/css" href="../css/common.css">
   <link rel="stylesheet" type="text/css" href="../css/<%= app %>.css">
 
-<% if (app === "applab") { %>
-  <link rel="stylesheet" type="test/css" href="../css/colpick.css">
-<% } %>
```

Comment:
Oh, I see it's used here:
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/_apps_dependencies.html.haml#L24
Not sure what for, though.  Might be worth tracking down.